index.bs

```diff
@@ -104,9 +104,10 @@ url: https://www.iso.org/standard/66067.html; spec: HEIF; type: property;
     text: image_height
     text: ster
     text: tmap
-
-url: https://www.iso.org/standard/66067.html; spec: HEIF; type: dfn;
     text: derived image item
+    text: aux_type
+    text: AuxiliaryTypeProperty
+    text: AuxiliaryTypeInfoBox
```

Comment:
Optional: Reorder these two lines if you want to list these in alphabetical order. We seem to be appending to the end of the list most of the time.